CONTRIBUTING.rst

```diff
@@ -66,3 +130,47 @@ Last thing
 
   The title of your PR will become the commit message, please craft it
   with care.
+
+How to make a new release
+-------------------------
+
+If you want to hit the ground running and publish a new release on a freshly set-up machine, the following should suffice:
```

Comment:
I'm not sure what you mean by "ground running" and why it is important to do that on a "freshly set-up machine".
Would the following sentence be sufficient enough?

```suggestion
If you want to publish a new release, use the following steps
```